### PR TITLE
Fiscal event aggregate

### PR DESCRIPTION
**Pull request by pintu-eGov**

### CODE REVIEW

File: core/libraries/ifix-services-common/pom.xml [hunk 1]
```diff
@@ -1,73 +1,73 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (rushang7-eGov) [inline core/libraries/ifix-services-common/pom.xml:1]: Why are there any changes to this file?

File: domain-services/fiscal-event-service/fiscal-event-service-0.1.0.yaml [hunk 2]
```diff
@@ -0,0 +1,374 @@
+---
```
Comment (rushang7-eGov) [inline domain-services/fiscal-event-service/fiscal-event-service-0.1.0.yaml:1]: Please remove any changes to the file outside of the aggregator project.

File: reference-dashboard/fiscal-event-aggregator/src/test/java/org/egov/ifix/aggregate/FiscalEventAggregatorServiceApplicationTests.java [hunk 3]
```diff
@@ -0,0 +1,13 @@
+package org.egov.ifix.aggregate;
```
Comment (rushang7-eGov) [inline reference-dashboard/fiscal-event-aggregator/src/test/java/org/egov/ifix/aggregate/FiscalEventAggregatorServiceApplicationTests.java:1]: Remove unused test file.

**Review by rushang7-eGov — CHANGES_REQUESTED** [review]

File: domain-services/fiscal-event-service/fiscal-event-service-0.1.0.yaml (hunk 2 above)
Comment (pintu-eGov) [reply domain-services/fiscal-event-service/fiscal-event-service-0.1.0.yaml:1]: haven't done any changes in this file. while taking pull from develop and got conflict and accepted the develop branch one.

File: core/libraries/ifix-services-common/pom.xml (hunk 1 above)
Comment (pintu-eGov) [reply core/libraries/ifix-services-common/pom.xml:1]: haven't done any changes in this file. while taking pull from develop and got conflict and accepted the develop branch one.

File: reference-dashboard/fiscal-event-aggregator/src/test/java/org/egov/ifix/aggregate/FiscalEventAggregatorServiceApplicationTests.java (hunk 3 above)
Comment (pintu-eGov) [reply reference-dashboard/fiscal-event-aggregator/src/test/java/org/egov/ifix/aggregate/FiscalEventAggregatorServiceApplicationTests.java:1]: done.

**Pull request closed without merging**